packages/resource-deployment/scripts/function-app-create.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+set -euo pipefail
+
+exitWithUsageInfo() {
+    echo "
+Usage: $0 -s <subscription name or id> -r <resource group> -l <location>
+"
+    exit 1
+}
+
+export subscription=""
+export resourceGroupName=""
+export location=""
+export functionAppName
+export appInsightsName
+export storageAccountName
+
+# Set default ARM Function App template files
+templateFilePath="../templates/function-app-template.json"
+
+# Read script arguments
+while getopts "s:r:l:"  option; do
+    case $option in
+		s) subscription=${OPTARG} ;;
+		r) resourceGroupName=${OPTARG} ;;
+		l) location=${OPTARG} ;;
+		*) exitWithUsageInfo ;;
+		esac
+done
+
+if [ -z "$subscription" ] || [ -z "$resourceGroupName" ] || [ -z "$location" ]; then
+	echo "Either one of subscription, resourceGroupName or location is empty"
+	exitWithUsageInfo
+fi
+
+# Login to Azure account if required
+if ! az account show 1>/dev/null; then
+    az login
+fi
+
+#set the default subscription id
+az account set --subscription $subscription
+
+#Check for existing RG
```

Comment:
remove this. let this script expect that resource group exists. use the same rg as we use for batch in install.sh